datacenterlight/templates/datacenterlight/includes/_calculator_form.html

```diff
@@ -1,4 +1,14 @@
 {% load staticfiles i18n%}
+{% if vm_pricing %}
+<script type="application/javascript">
+    window.vat_inclusive = {% if vm_pricing.vat_inclusive %}true{% else %}false{% endif%};
+    window.vat_percentage = {% if vm_pricing.vat_percentage %}{{vm_pricing.vat_percentage}}{% else %}0{% endif%};
+    window.coresUnitPrice = {% if vm_pricing.cores_unit_price %}{{vm_pricing.cores_unit_price}}{% else %}0{% endif%};
+    window.ramUnitPrice = {% if vm_pricing.ram_unit_price %}{{vm_pricing.ram_unit_price}}{% else %}0{% endif%};
+    window.ssdUnitPrice = {% if vm_pricing.ssd_unit_price %}{{vm_pricing.ssd_unit_price}}{% else %}0{% endif%};
+    window.hddUnitPrice = {% if vm_pricing.hdd_unit_price %}{{vm_pricing.hdd_unit_price}}{% else %}0{% endif%};
+</script>
+{% endif %}
```

Comment:
initializing globals in the plugin template will cause the `vm_pricing` for the last loaded calculator to override the values for any other calculators on the page. So, to avoid the pitfall, I suggest the `vm_pricing` values can be added as data attributes to respective inputs, maybe suited as a separate task.

Reply:
Valid point. I did not think about "more than one calculator in a page" scenario.